utils/spacewalk-common-channels.ini

```diff
@@ -1328,6 +1328,38 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_leap16_0]
+checksum = sha256
+archs    = x86_64, aarch64, s390x, ppc6le
+name     = openSUSE Leap 16.0 (%(arch)s)
+gpgkey_url = file:///usr/lib/rpm/gnupg/keys/gpg-pubkey-29b700a4-62b07e22.asc
+gpgkey_id = 29B700A4
+gpgkey_fingerprint = AD48 5664 E901 B867 051A B15F 35A2 F86E 29B7 00A4
+repo_url = https://cdn.opensuse.org/distribution/leap/16.0/repo/oss/
+dist_map_release = 16.0
+
+# This is expected. openSUSE Leap 16.0 client tools are valid for all openSUSE Leap 16.X releases
+[opensuse_leap16_0-uyuni-client]
+name     = Uyuni Client Tools for %(base_channel_name)s
+archs    = x86_64, aarch64, s390x, ppc6le
+base_channels = opensuse_leap16_0-%(arch)s
+checksum = sha256
+gpgkey_url = %(_uyuni_gpgkey_url)s
+gpgkey_id = %(_uyuni_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
+repo_url = https://cdn.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/SLMicro6-Uyuni-Client-Tools/SL-Micro6/
```

Comment:
Work for now, but we need to decide if we want SLMicro6 or other client tools. Should be related to what we want to do for SUSE Linux 16 as well.